src/main/java/net/earthcomputer/clientcommands/command/AliasCommand.java

```diff
@@ -0,0 +1,186 @@
+                .then(literal("list")
+                        .executes(ctx -> listAliases()))
+                .then(literal("remove")
+                        .then(argument("key", string())
+                                .executes(ctx -> removeAlias(getString(ctx, "key"))))));
+
+        for (String key: aliasMap.keySet()) {
+            if (dispatcher.getRoot().getChildren().stream().map(CommandNode::getName).noneMatch(literal -> literal.equals(key))) {
+                addClientSideCommand(key);
+                dispatcher.register(literal(key)
+                        .executes(ctx -> executeAliasCommand(key, null))
+                        .then(argument("arguments", greedyString())
+                                .executes(ctx -> executeAliasCommand(key, getString(ctx, "arguments")))));
+            } else {
+                LOGGER.error("Attempted to register alias /{}, but that command already exists", key);
+            }
+        }
+    }
+    private static int executeAliasCommand(String aliasKey, String arguments) throws CommandSyntaxException {
+
+        String cmd;
+        if (aliasMap.containsKey(aliasKey)) {
+            cmd = aliasMap.get(aliasKey);
+        } else {
+            throw NOT_FOUND_EXCEPTION.create(aliasKey);
+        }
+        int inlineArgumentCount = (int) Pattern.compile("%[^%]").matcher(cmd).results().count();
+        if (inlineArgumentCount > 0) {
+            String[] argumentArray = arguments.split(" ", inlineArgumentCount + 1);
+
+            String trailingArguments = "";
+            if (argumentArray.length > inlineArgumentCount) {
+                trailingArguments = " " + argumentArray[inlineArgumentCount];
+            }
+            try {
+                cmd = String.format(cmd, (Object[]) argumentArray) + trailingArguments;
+            } catch (Exception e) {
+                LOGGER.error(e);
+                throw TOO_FEW_ARGUMENTS_EXCEPTION.create();
+            }
+        } else if (arguments != null){
+            cmd += " " + arguments;
+        }
+        assert MinecraftClient.getInstance().player != null;
+        MinecraftClient.getInstance().player.sendChatMessage(cmd);
+
+        return 0;
+    }
+
+    private static int addAlias(String key, String command) throws CommandSyntaxException {
+        ClientPlayNetworkHandler networkHandler = MinecraftClient.getInstance().getNetworkHandler();
+        assert networkHandler != null;
+        var dispatcher = (CommandDispatcher<ServerCommandSource>) (CommandDispatcher<?>) networkHandler.getCommandDispatcher();
+
+        if (aliasMap.containsKey(key)) {
+            throw ALIAS_EXISTS_EXCEPTION.create(key);
+        }
+        if (dispatcher.getRoot().getChildren().stream().map(CommandNode::getName).anyMatch(literal -> literal.equals(key))) {
+            throw COMMAND_EXISTS_EXCEPTION.create(key);
+        }
+        if (command.charAt(0) != '/') {
```

Comment:
```suggestion
        if (!command.startsWith("/")) {
```
this also handles the case where `command` is empty